catalog/app/containers/Auth/messages.js

```diff
@@ -30,14 +30,44 @@ export default defineMessages({
     defaultMessage: 'Authenticating...',
   },
 
+  // SSO
+  ssoGoogleUse: {
+    id: `${scope}.SSO.Google.use`,
+    defaultMessage: 'Use Google Account',
+  },
+  ssoGoogleError: {
+    id: `${scope}.SSO.Google.error`,
+    defaultMessage: "Couldn't sign in with your Google account",
+  },
+  ssoGoogleNotFound: {
+    id: `${scope}.SSO.Google.notFound`,
+    defaultMessage: "There's no user linked to your Google account",
+  },
+  ssoSignUpHeading: {
+    id: `${scope}.SSO.SignUp.heading`,
+    defaultMessage: 'Complete sign-up',
```

Comment:
not sure where this appears and what user is supposed to do? Does this mean they need to finish sign up or...?

Reply:
this is a heading for the sso sign-up form (displayed when a person uses sso, but doesn't have quilt account yet), it has username and (when enabled) password prompts

Reply:
OK it should be `Complete sign up`